app-vcs/brz/autobuild/defines

```diff
@@ -0,0 +1,16 @@
+PKGNAME=brz
```

Comment:
Based on the source name, this package is better named `breezy`. You could also add `PKGPROV=brz`.